src/sentry/api/bases/organization_events.py

```diff
@@ -60,6 +67,12 @@ def get_team_ids(self, request: Request, organization: Organization) -> Sequence
 
         return [team.id for team in teams]
 
+    def get_dataset(self, request: Request) -> Any:
```

Comment:
💢 really annoyed with this, unfortunately can't do:
```suggestion
    def get_dataset(self, request: Request) -> Union[discover, metrics_enhanced_performance]:
```